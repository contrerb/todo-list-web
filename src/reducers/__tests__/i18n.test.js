import reducer, { actions } from "../i18n";

it("returns the initial state", () => {
  expect(reducer(undefined, {})).toMatchSnapshot();
});

it("handles I18N_LANGUAGE", () => {
  expect(reducer(undefined, actions.setLanguage("lang"))).toMatchSnapshot();
});

it("handles I18N_LOCALE", () => {
  expect(reducer(undefined, actions.setLocale("locale"))).toMatchSnapshot();
});
