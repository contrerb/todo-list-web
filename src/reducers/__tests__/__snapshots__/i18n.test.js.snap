// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`handles I18N_LANGUAGE 1`] = `
Object {
  "language": "lang",
  "locale": "en-GB",
}
`;

exports[`handles I18N_LOCALE 1`] = `
Object {
  "language": "en-GB",
  "locale": "locale",
}
`;

exports[`returns the initial state 1`] = `
Object {
  "language": "en-GB",
  "locale": "en-GB",
}
`;
